include/seqan3/alignment/pairwise/alignment_configurator.hpp

```diff
@@ -295,6 +295,20 @@ struct alignment_configurator
     template <typename function_wrapper_t, typename ...policies_t, typename config_t>
     static constexpr function_wrapper_t configure_free_ends_optimum_search(config_t const & cfg);
 
+    /*!\brief Determines the trace type.
+     * \tparam config_t The configuration type.
+     */
+    template <typename config_t>
+    struct configure_trace_type
+    {
+        //!\brief If traceback is enabled resolves to seqan3::detail::trace_directions,
+        //!\      otherwise seqan3::detail::ignore_t.
```

Comment:
Why not std::ignore?

Reply:
std::ignore is not a type but a variable. ignore_t is `remove_cv_t<decltype(std::ignore)>`